scripts/plopfile.js

```diff
@@ -15,8 +15,7 @@ let rspProject = 'React Spectrum v3';
 let otherProject = 'other';
 
 module.exports = function (plop) {
-  plop.setActionType('renameMany', require('../plop-actions/renameMany'));
-
+  
```

Comment:
if there's no more references to renameMany, we should delete it in this PR as well

Reply:
Deleted the renameMany.js file and signed the CLA :)